docker_explorer/de.py

```diff
@@ -156,14 +153,23 @@ def GetAllContainers(self):
 
     Returns:
       list(Container): the list of Container objects.
+
+    Raises:
+      errors.BadStorageException: If required files or directories are not found
+        in the provided Docker directory.
     """
+    if not os.path.isdir(self.containers_directory):
+      raise errors.BadStorageException(
+          'Containers directory {0} does not exist'.format(
+              self.containers_directory))
     container_ids_list = os.listdir(self.containers_directory)
     if not container_ids_list:
-      print('Couldn\'t find any container configuration file (\'{0:s}\'). '
-            'Make sure the docker repository ({1:s}) is correct. '
+      print('Could not find container configuration files ({0:s}) in {1:s}.\n'
+            'Make sure the docker repository ({2:s}) is correct.\n'
```

Comment:
Sounds like you mean `docker directory` instead of `repository`?

Reply:
Fixed